torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -212,8 +238,27 @@ std::pair<IValue, c10::optional<IValue>> getFunctionTuple(
   // Make a copy of the constants and append the method names
   // that we emitted for the converted INTERFACE_CALL nodes above.
   auto constants = code.constant_table();
+  std::vector<IValue> deduplicated_constants;
+  for (const auto& constant : constants) {
+    if (constant.isTensor()) {
+      const auto& constant_tensor = constant.toTensor();
+      if (constants_from_jit.find(constant_tensor) ==
+          constants_from_jit.end()) {
+        deduplicated_constants.emplace_back(constant);
```

Comment:
You can refactor this with:
```
auto it = constants_from_jit.find(constant_tensor)l
if (it == constants_from_jit.end()) {
...
} else {
auto index = IValue(*it);
}
```
It helps reduce extra lookup when doing `constants_from_jit.at`. Totally optional though. This is not perf critical part anyway but just as coding practice. I learned this one from @dreiss.

Reply:
@kimishpatel There's also `insert()` which ignores the insert request if the element already exists. This effectively does what the code code about is trying to do!

https://en.cppreference.com/w/cpp/container/unordered_map/insert

> Returns a pair consisting of an iterator to the inserted element (or to the element that prevented the insertion) and a bool denoting whether the insertion took place.